tools/meteor-services/auth.js

```diff
@@ -65,6 +65,9 @@ var loggedInAccountsConnection = function (token) {
     connection.close();
 
     if (err.error === 403) {
+      // Better than saying nothing at all
+      Console.error("warning: login token was rejected by the server");
```

Comment:
> The solution was to simply regenerate the token

Ok, so the solution was needing to re-`meteor login`, generate a new `METEOR_SESSION_FILE` before doing a `meteor deploy`?  Just trying to figure out where this error was getting thrown.  Plus, I'm mainly curious if we should make this error actually tell them how to fix the problem (i.e. Find something/add something to docs and link to it in the actual error).

You can just push new commits to the same branch (`LegalRobot/meteor-tool-auth-messaging`) and it will update this PR. :)